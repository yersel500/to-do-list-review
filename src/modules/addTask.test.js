import ArrayTask from "./class-methods";



describe('Test addTask function', () => {
  const arraytask = new ArrayTask();
  test('Add the object task and check if it works',() => {
    const insertTask = {
      description: 'task1',
      complete: false,
      index: 0
    }

    arraytask.addTask(insertTask);
    expect(arraytask).toMatchSnapshot();
  })
})