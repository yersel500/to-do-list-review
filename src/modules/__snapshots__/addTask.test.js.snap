// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test addTask function Add the object task and check if it works 1`] = `
ArrayTask {
  "tasks": Array [
    Object {
      "complete": false,
      "description": "task1",
      "index": 0,
    },
  ],
}
`;
